docs/zh/latest/architecture-design/plugin.md

```diff
@@ -26,8 +26,8 @@ title: Plugin
 `Plugin` 配置可直接绑定在 `Route` 上，也可以被绑定在 `Service` 或 `Consumer`上。而对于同一
 个插件的配置，只能有一份是有效的，配置选择优先级总是 `Consumer` > `Route` > `Service`。
 
-在 `conf/config.yaml` 中，可以声明本地 APISIX 节点都支持哪些插件。这是个白名单机制，不在该
-白名单的插件配置，都将会被自动忽略。这个特性可用于临时关闭或打开特定插件，应对突发情况非常有效。
+在 `conf/config.yaml` 中，可以声明本地 APISIX 节点都支持哪些插件。这是个白名单机制，不在该白名单的插件配置，
```

Comment:
The generated result doesn't have this problem.
https://apisix.apache.org/zh/docs/apisix/architecture-design/plugin/
<img width="516" alt="微信截图_20211216091702" src="https://user-images.githubusercontent.com/4161644/146290153-7319d9c4-5699-4458-87f4-d5a7a1a7d962.png">

Reply:
Look like it is the problem of apisix-website in some browsers?
@juzhiyuan 
Could you take a look?

Reply:
<img width="855" alt="微信截图_20211216092955" src="https://user-images.githubusercontent.com/4161644/146291266-099dd685-acc7-411e-80ae-48f13cd95a43.png">

The display on GitHub is OK.

Reply:
![image](https://user-images.githubusercontent.com/2106987/146302923-9cca6f64-8f96-42b5-8566-59f6f16e1a2e.png)

Hi @spacewander, it's more like a Browser issue 🤔 I just checked the website & GitHub on Chrome, it has this extra space, too. Maybe that's because of the breaking line in markdowns 😄 

![image](https://user-images.githubusercontent.com/2106987/146303098-55b428b9-2f00-4272-9fd8-196d98a47e80.png)

Thanks for @Nidaozi's PR and @spacewander's check, we could fix this **style** issue by this PR in my opinion, but need @Nidaozi's help to update your PR's title and description :)

Reply:
I have confirmed this issue with others who're using Chrome in windows.

Reply:
thx; I have changed

Reply:
```
`Plugin` 配置可直接绑定在 `Route` 上，也可以被绑定在 `Service` 或 `Consumer`上。而对于同一
个插件的配置，只能有一份是有效的，配置选择优先级总是 `Consumer` > `Route` > `Service`。
```

also have the same problem. Is it possible to add a way to detect this pattern?

Reply:
This problem is introduced when the browser is parsed. There are two ways to solve this problem. The first method uses regular detection to replace ‘\n’ with empty when it detects that the end is Chinese and the next line is still Chinese. The second way is to modify the CSS rendered by Markdown, set the'font-size' of the parent tag to 0 and the'font-size' of the child tag to the desired font size. https://stackoverflow.com/questions/8550112/prevent-browser-converting-n-between-lines-into-space-for-chinese-characters/8551033#8551033

Reply:
Get it, I would prefer to seek help from Docusaurus 😄 Hi @slorber @Josh-Cena, would you please take look at this issue?

Reply:
Hey, it's indeed a Markdown/HTML feature that they coerce line breaks to spaces. Generally we don't recommend prose wrapping for exactly this reason (and also for the sake of cleaner git diffs). You can try it in the MDX playground: https://mdx-git-renovate-babel-monorepo-mdx.vercel.app/playground

![image](https://user-images.githubusercontent.com/55398995/146316200-79738877-be8e-41d0-888e-1a1e7ff0c9df.png)

Maybe Firefox doesn't do that? I've tested with Chrome and Safari and they both create the spaces

Reply:
![image](https://user-images.githubusercontent.com/2106987/146317205-3ffd9444-8e63-4b67-aeb1-6a222bd88262.png)

![image](https://user-images.githubusercontent.com/2106987/146317408-4dae99a6-38c9-4350-894a-a4eec4a4111d.png)

Thanks to @Josh-Cena's clarification, after testing on the playground: If we use line-breaking or one space, it will be rendered in one line with a space; If we use 2 spaces and line breaking, it will be rendered in 2 lines.

If we need to fix this style rendered issue, we have to remove all line-breakings from markdown files. @Nidaozi, could you help to check if we have many cases like this? I would hope we can fix this case in one PR, but don't want it to be very lard.

Reply:
> If we use 2 spaces and line breaking, it will be rendered in 2 lines.

It's actually part of the Markdown spec that two trailing spaces => `<br />` tag and two line breaks => begin a new paragraph.

Reply:
🙈🙈 How about closing this PR? 

cc @Nidaozi

Reply:
ok